src/backend/distributed/metadata/metadata_cache.c

```diff
@@ -2506,6 +2507,17 @@ DistPlacementPlacementidIndexId(void)
 }
 
 
+/* return oid of pg_dist_colocation_pkey */
+Oid
+DistColocationIndexId(void)
```

Comment:
Could you please make colocation index related code a separate commit? It is not so easy to understand that we've added them for the sake of test consistency now.

Reply:
makes sense. I'll do that once the PR is approved